src/views/v2/components/events-bar/search/keyword.php

```diff
@@ -29,7 +29,23 @@ class="tribe-common-form-control-text__input tribe-events-c-search__input tribe-
 		id="tribe-events-events-bar-keyword"
 		name="tribe-events-views[tribe-bar-search]"
 		value="<?php echo esc_attr( tribe_events_template_var( [ 'bar', 'keyword' ], '' ) ); ?>"
-		placeholder="<?php esc_attr_e( 'Search for events', 'the-events-calendar' ); ?>"
-		aria-label="<?php printf( esc_attr__( 'Enter Keyword. Search for %s by Keyword.', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?>"
+		placeholder="<?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 835367c59227983ac351c6130b10e595</sub>